Lib/pathlib.py

```diff
@@ -751,7 +751,7 @@ def cwd(cls):
         """Return a new path pointing to the current working directory
         (as returned by os.getcwd()).
         """
-        return cls(os.getcwd())
+        return cls().absolute()
```

Comment:
Same remark as other PR about feeling weird about a straightforward implementation matching docstring being replaced by an indirect implementation.

Could you add a note in `absolute` to mark that `cwd` depends on `absolute` using `os.getcwd`?

Reply:
I've added a comment in `cwd()` and adjusted its docstring. Strictly speaking `cwd` _does not_ depend on `absolute` using `os.getcwd`. It relies on `absolute` returning an absolute path per the POSIX definition (4.11 [here](https://pubs.opengroup.org/onlinepubs/009695399/basedefs/xbd_chap04.html)). We may change the implementation of `absolute()` to use `nt._getfullpathname()` on Windows in future, but this would not break `cwd()`.

Reply:
The comment is great.  Thank you for all your pathlib efforts!